examples/motoko_examples/simple-to-do/canisters/azle/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Opt, Query, Update, ic, nat, nat64 } from 'azle';
+import { ToDo } from './types';
+
+import * as Utils from './utils';
```

Comment:
This whole utils module is not in the Motoko code, I think we should get rid of it and make the Azle code match the Motoko code.